### PR TITLE
Update lastGameActivity upon focus

### PR DESCRIPTION
**Pull request by schoentoon**
So, the lobby timer in afk warden wasn't working for me at all. I assume the following is supposed to be called upon focus
```javascript
instance.game_activity_signal.connect(function(){
   lastGameActivity = window.performance.now();
});
```
Upon some investigating however it seems like this isn't being called, or at least not reliably. By updating from the callback a bit higher up (as seen in the pull request) it does seem to be updated more reliably.

### CODE REVIEW

**Comment by whs** [comment]
The two signals here has different meaning

- `game_activity_signal` - It supposed to fire when there is activity in the game window
  - In Alt1, due to keylogging concerns (it is not open source, although it's not obfuscated either) it only hook mouse events. In most activities in RuneScape you would click more than typing a keyboard anyway
  - In RuneKit I believe the Mac version has both mouse and keyboard hooked, but in Linux I can only hook the keyboard without using XInput2 (which has incomplete binding in python-xcb). If we try to passively grab the mouse then the input won't passthrough to the game window. (I believe the use case here is for global hotkeys, and it is possible that my passive mouse grab wasn't implemented properly - the spec is really hard to read)
  - So in summary in Linux you'd have to type something to count as game activity, unlike on Windows where you have to click
- `game_active_change_signal` - This supposed to fire on window focus/blur

I think this change is a good hack, but I'd rather prefer to fix mouse hooking. Also I believe focus/blur event wouldn't count against lobby timer, but keyboard or mouse event would.

**Comment by schoentoon** [comment]
Interesting, that explains why it sometimes seemed to work (I guess I pressed a key everytime it did work). And yeah you are correct, focus isn't equal to resetting the ingame lobby timer. But I personally always right click the moment I focus anyway, so I'll keep running with this little hack for now. While looking for a better solution for this.
